core-concepts/dynamic-ui/application-layout.md

```diff
@@ -0,0 +1,20 @@
+---
+description: The Application layout can be selected to match the application size to the device it is intended for.
+---
+
+# Application Layout
+
+## Selecting Application Layout
+
+Default Device layout is set by default for an application. To change it, click on the Layout control present on the top of the canvas and select a layout to use.
```

Comment:
Here you should talk about at a high level how does the feature work i.e each device type has a min/max range within which the UI scales. Sizes smaller / larger than this will cause scrolling / additional whitespace so choose your device type carefully. Device types can be changed on the fly. etc etc.